deploy/addon/matomo.md

```diff
@@ -0,0 +1,119 @@
+3. In your terminal, you can then run `clever addon create jenkins <app-name> --region <region> --org <org>` where `app-name` is the name you want for your add-on, `region` deployment region, and `org` the organization ID the application will be created under.
+
+Refer to the [documentation]({{< ref `/reference/clever-tools/create.md` >}}) for more details on application creation with Clever Tools
+
+## Accessing the Matomo interface
+
+Once you created your add-on, you should get to the dashboard. You should see a link named `Access Matomo`. Opening that link will redirect you to our SSO authentication.
+
+{{< readfile `/content/partials/single-sign-on.md` >}}
+
+## Configure your Matomo instance
+
+Once you accessed your Matomo interface, we can start configuring it. A custom Clever Cloud configuration is automatically installed on your instance during the provisioning.
+
+This configuration helps you taking advantage of optimisations by using a Redis cache by default. 
+
+### Using your Matomo
+
+If you're new with Matomo, you would probably want to read the Matomo's guides on [the official documentation](https://matomo.org/guides/)
+
+## Customize your Matomo instance
+
+Matomo can be customized following your needs with a multitude of plugins. You can go into `Settings` gear icon and then `Marketplace` to manage them. Ultimately, you can click Install on whatever plugin you could find accurate.
+
+Plugins can be browsed from [Matomo own plugin repository](https://plugins.matomo.org/).
+
+## Security and updates
+
+The Matomo addon is a managed application in the way you don't have to select a particular version. Still its receives updates for both features and security, that we will managed for you with continuously upgraded version over time.
+
+After having been updated, Matomo could need need to restart.
+
+
+## Plans
+
+If Matomo comes with the easiest path to be installed, you could want to adjust the flavour for your instance or database. Here are the plans for PHP instances as well as its MySQL and Redis addons.
+
+<!-- Documentation is here: https://www.clever-cloud.com/doc/clever-components/?path=/docs/%F0%9F%9B%A0-pricing-cc-pricing-product--default-story -->
+<script type="module" src="https://components.clever-cloud.com/load.js?version=7&components=cc-pricing-product.smart-addon,cc-pricing-product.smart-runtime"></script>
+
+### Matomo PHP Instances
+
+Those are the plans of the PHP instance.
+<div>
+<cc-smart-container context='{"productId": "PHP", "addonFeatures": ["cpu", "memory", "disk-size", "has-logs", "has-metrics"]}'>
+  <cc-pricing-product mode="addon" action="none">
+    <div slot="head"></div>
+  </cc-pricing-product>
+</cc-smart-container>
+</div>
+
+### MySQL database
+Those plans are the available databases flavours.
+<div>
+<cc-smart-container context='{"productId": "mysql"}'>
+  <cc-pricing-product mode="runtime" action="none" temporality='[{"type":"minute","digits":5}, {"type": "1000-minutes","digits": 2}]'>
```

Comment:
* We can remove the `temporality` attribute (this will fallback on day and 30 days)